tests/e2e/create.go

```diff
@@ -347,3 +352,111 @@ func reclaimPolicySynced(tc *v1alpha1.TidbCluster) (bool, error) {
 
 	return true, nil
 }
+
+func metaSynced(tc *v1alpha1.TidbCluster) (bool, error) {
+	ns := tc.GetNamespace()
+	tcName := tc.GetName()
+
+	pdControl := controller.NewDefaultPDControl()
+	pdCli := pdControl.GetPDClient(tc)
+	cluster, err := pdCli.GetCluster()
+	if err != nil {
+		logf(err.Error())
+		return false, nil
+	}
+	clusterID := strconv.FormatUint(cluster.Id, 10)
+
+	labelSelector := label.New().Cluster(tcName)
+	podList, err := kubeCli.CoreV1().Pods(ns).List(
+		metav1.ListOptions{
+			LabelSelector: labels.SelectorFromSet(
+				labelSelector.Labels(),
+			).String(),
+		},
+	)
+	if err != nil {
+		logf(err.Error())
+		return false, nil
+	}
+
+outerLoop:
+	for _, pod := range podList.Items {
+		podName := pod.GetName()
+		Expect(pod.Labels[label.ClusterIDLabelKey]).To(Equal(clusterID))
+
+		component := pod.Labels[label.ComponentLabelKey]
+		switch component {
+		case label.PDLabelVal:
+			var memberID string
+			members, err := pdCli.GetMembers()
+			if err != nil {
+				logf(err.Error())
+				return false, nil
+			}
+			for _, member := range members.Members {
+				if member.Name == podName {
```

Comment:
What if there is no member name matching pod name?

Reply:
Then this case will fail: https://github.com/pingcap/tidb-operator/pull/164/files/b036f31f8548f392e2e94579a8cb896f05b0a9f4#diff-c1c833494c542d5b15d71a65d557b44bR402

Reply:
done